local/bin/py/build/actions/pull_and_push_folder.py

```diff
@@ -80,13 +86,10 @@ def pull_and_push_folder(content, content_dir):
             if file_name.endswith("README.md"):
                 file_name = "_index.md"
         # Writing the new content to the documentation file
-        dirp = "{}{}".format(
-            content_dir,
-            content["options"]["dest_dir"][1:],
-        )
-        makedirs(dirp, exist_ok=True)
+        dirp = (Path(content_dir) / Path(content["options"]["dest_dir"].lstrip('/')) / directory).resolve()
+        dirp.mkdir(exist_ok=True)
```

Comment:
@davidejones  if we use `mkdir` instead of `makedirs` wouldnt the script throw a `FileNotFoundError` when trying to create possible nested directories under, for instance, `static_analysis/rules`: eg: `static_analysis/rules/python-best-practices/some-other-dir/some-markdown.md`?

Reply:
@StefonSimmons So it should work because mkdir on a pathlib object is the same as doing os.makedirs() 

Here is an example showing the equivalents https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module